### ISSUE TITLE
Current content review and updates

### ISSUE THREAD

**Issue by r-bartlett-gsa** [GSA/usagov-benefits-eligibility#551]
Review "Death of a loved one" life event content.

Update the following:

- [x] Start here: you vs person applying for benefits ( one option would be to change "you" to "applicant"; test with users #555 )
- [x] See #255 
- [x] Any additional revisions (working session with Meghan on 12/7/22), see https://designsystem.digital.gov/patterns/complete-a-complex-form/establish-trust/

**Comment by r-bartlett-gsa** [comment]
@maomeara63 It was brought up that it is not clear who is "you" in our criteria questions for example: "Your date of birth..." or "Your relationship to the person who died...". In some places we add "you or the person who's applying for benefits", but not all. In reviewing this, I don't think we should update all of the criteria to say "you or the person..." It seems repetitive and too wordy. Let's test this with users, give them the scenario that they are looking for benefits for a friend or a family member and not themselves and see if they get confused.

**Comment by maomeara63** [comment]
@r-bartlett-gsa This is a good call out. I think the questions were clearer before we moved the gating questions to the top. Looking at it now, I'm not sure if the very first gating question is about the decedent or the person completing the form. I think this might be part of a larger review of the questions. For example, in asking about the person completing the form... is the marital status question re marriage to the decedent? (If I am a friend helping someone to complete the form, do you really care about my marital status?) Let's meet and talk through the content prior to testing.
